beaker-hcloud.gemspec

```diff
@@ -21,6 +21,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rubocop', '~> 1.25'
   s.add_development_dependency 'rubocop-rake', '~> 0.6'
   s.add_development_dependency 'rubocop-rspec', '~> 2.9'
+  s.add_runtime_dependency 'bcrypt_pbkdf', '~> 1.0'
   s.add_runtime_dependency 'beaker', '~> 4.38'
+  s.add_runtime_dependency 'ed25519', '~> 1.2'
   s.add_runtime_dependency 'hcloud', '>= 1.0.3', '< 2.0.0'
+  s.add_runtime_dependency 'ssh_data', '~> 1.3'
```

Comment:
a bit dissapointing that the ed25519 isn't sufficient here

Reply:
Well, the `ed25519` gem is only about the algorithm itself. It knows nothing about SSH. The (to me) disappointing bit was that no existing gem seems to support the key generation / serialization of ed25519 keys. `net-ssh` does not include key generation at all. There is a gem called `sshkey` which would be perfect but only supports RSA/DSA (with partial support for _reading_ ed25519 keys).

`ssh_data` was the closest thing I could find for what we need, but it still lacked the serialization of the private key.